cns/api.go

```diff
@@ -365,3 +365,8 @@ type EndpointRequest struct {
 	HostVethName  string `json:"hostVethName"`
 	InterfaceName string `json:"InterfaceName"`
 }
+
+type GetVMUniqueIDResponse struct {
+	Response   Response `json:"response"`
+	VMUniqueID string   `json:"VMUniqueID"`
```

Comment:
very nitpicky so feel free to ignore but my brain really wants consistent casing in the json field names, e.g.
```json
{
  "response": {...},
  "vmUniqueID": "..."
}
```

Reply:
we should at least be self-consistent. I feel like the convention when interacting with JSON in Go is to lowercase the fields

Reply:
I kept it this way due to acronyms like VM or ID etc. Existing structs have it this way. But let me lowercase it all to follow the Go convention.